src/libexpr/value.hh

```diff
@@ -38,6 +39,7 @@ class XMLWriter;
 
 
 typedef long NixInt;
+typedef double NixFloat;
```

Comment:
One problem: using a double here will interfere with #597 (using 2 words instead of 3 for Values) on 32-bit machines. A float might be better, at least on 32-bit...